benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp_ompss/nemolite2d.f90

```diff
@@ -0,0 +1,156 @@
+program gocean2d
+  use dl_timer, only: timer_start, timer_stop, timer_init, timer_report, i_def64
+  use grid_mod
+  use field_mod
+  use initialisation_mod, only: initialisation
+  use model_mod
+  use gocean2d_io_mod, only: model_write
+  use gocean_mod,      only: model_write_log, gocean_initialise, &
+                             gocean_finalise
+  !use likwid
```

Comment:
I think we can get rid of the likwid calls/usage from this file.

Reply:
Done - The main C++ code may also have these calls as I copied from there @sergisiso

Reply:
tbh the algorithm layer file should be an exact copy in all psykal_* implementations, we can consider (in another PR) that all Makefiles use the same source file or that each folder have a symbolic link to a copy on the parent folder.

Reply:
Yes, I was thinking the same @sergisiso but, as you say, that's not something for this PR. (We could also add a profiling wrapper for the likwid API.)